utils/physics.js

```diff
@@ -0,0 +1,59 @@
+import Matter from "matter-js";
+import { getPipeSizePosPair } from "./randomLogic";
+import { Dimensions } from "react-native";
+
+const { height: windowHeight, width: windowWidth } = Dimensions.get("window");
```

Comment:
I added this here to show you how can interpolate values and reassign them in a cleaner manner, there are a few other places in your codebase you can make this change